### PR TITLE
Update alpha.json

### PR DESCRIPTION
**Pull request by YouFoundAlpha**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] You have completed your website. 
- [x] The website is reachable.  
- [x] You're not using Vercel or Netlify.  
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [ ] There is sufficient information at the `owner` field.  

## Website Link/Preview

https://yfaweb.pages.dev

### CODE REVIEW

**Comment by YouFoundAlpha** [comment]
Checks say that the domain is invalid but it is valid

File: domains/alpha.json [hunk 1]
```diff
@@ -4,8 +4,8 @@
         "email": "sweepalf@gmail.com"
     },
     "record": {
-        "A": ["217.174.245.249"],
-        "MX": ["hosts.is-a.dev"],
-        "TXT": ["v=spf1 a mx ip4:217.174.245.249 ~all"]
+        "CNAME": "yfaweb.pages.dev",
```
Comment (DEV-DIBSTER) [inline domains/alpha.json:7]: Cannot have a CNAME in conjunction with other records.

**Review by DEV-DIBSTER — DISMISSED** [review]

**Comment by YouFoundAlpha** [comment]
@DEV-DIBSTER I've changed it

File: domains/alpha.json (hunk 1 above)
Comment (YouFoundAlpha) [reply domains/alpha.json]: Resolved.

**Review by wdhdev — APPROVED** [review]

**Comment by YouFoundAlpha** [comment]
@WilliamDavidHarrison Does is-a.dev support more than one TXT record?

**Comment by YouFoundAlpha** [comment]
Does is-a.dev support more than one TXT record on a subdomain?

**Pull request merged**